soc/bouffalolab/bl6/Kconfig.defconfig

```diff
@@ -8,6 +8,12 @@ config SYS_CLOCK_HW_CYCLES_PER_SEC
 	default 192000000
 
 config NUM_IRQS
-	default 64
+	default 80
```

Comment:
again, changing things you just added because they're wrong, whole PR needs reworking